README.rst

```diff
@@ -75,15 +71,13 @@ which is default) in one terminal first:
 
 .. code-block:: shell
 
-    $ docker-compose up web
+    make runkinto
```

Comment:
Nit: in kinto-http.py we have `make runkinto` https://github.com/Kinto/kinto-http.py/blob/bff433440a1eb468ff4d336349aada20fb73b357/Makefile#L32
but in kinto-attachment we have `make run-kinto` https://github.com/Kinto/kinto-attachment/blob/2d58f8a341073602089f97087c792b0eea836014/Makefile#L53

Take advantage of this to choose the one you prefer and we align them all :)

Reply:
https://github.com/Kinto/kinto-http.py/pull/272